client/sites-dashboard/components/sites-table-row.tsx

```diff
@@ -166,7 +167,9 @@ export default function SitesTableRow( { site }: SiteTableRowProps ) {
 						/>
 					</Column>
 					<Column mobileHidden>{ site.plan.product_name_short }</Column>
-					<Column mobileHidden>July 16, 1969</Column>
+					<Column mobileHidden>
+						{ site.options?.updated_at ? <TimeSince date={ site.options.updated_at } /> : '' }
```

Comment:
Even this would probably work. Pretty sure React is happy to render `undefined` values as nothing.

```suggestion
						{ site.options?.updated_at && <TimeSince date={ site.options.updated_at } /> }
```

Reply:
I've tested this too, and it worked fine. I saw that we use both ways in the codebase, but I used one with an explicitly empty string as it seemed cleaner. Is one of those ways preferred over another?

Reply:
No I don't think so. I think `&&` is pretty common around the place but I don't think it's any better tbh.